docs/how-to/account-info.md

```diff
@@ -2,13 +2,13 @@
 
 When you connect your PMM instances to Percona Platform, PMM gets access to:
 
--  more alert templates 
+- [More alert templates](../get-started/alert-templates.md)
 -  Basic Advisor Checks for additional database checks
 -  Standard/Premium Advisor Checks for more advanced database health checks.
```

Comment:
Since with this PR we also removed Standard/Premium reference from other pages, we can consider to do the same from this account-info page as well, to remove possible confusion? Standard/Premium is also referenced below.  We can also consider to link the advisor detail page directly in this bullet list, like we do for the additional alert templates (just for consistency - currently the link is below the list)?

Reply:
@sleto-it , I will update the Standard/Premium references with a separate PR. I have an[ open task for this](https://perconadev.atlassian.net/browse/PMM-13299?atlOrigin=eyJpIjoiNDg5YmVkYTRlMTA0NGMwZjg4ODFlMzE5ZDA0OGUzMzEiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) and will get to it soon